_posts/2023-09-15-kafka-producer-client-essentials.md

```diff
@@ -0,0 +1,394 @@
+---
+layout: post
+title: "Developing Kafka clients: A simple producer application"
```

Comment:
Should we call it `Developing Kafka client applications: A simple producer` or something like that as discussed in https://github.com/strimzi/strimzi.github.io/pull/384#discussion_r1319713209